run.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# This script is used to generate Go and Python language bindings for the proto files in this repository.
+# It uses ghcr.io/srl-labs/protoc container image to generate the bindings. See https://github.com/srl-labs/protoc-container for more details.
+# to generate all protos: ./run.sh compile-all
+# to generate a specific proto for Go: ./run.sh compile-go-gnmi
+# to generate a specific proto for Python: ./run.sh compile-py-gnmi
+
+
+set -o errexit
+set -o pipefail
+
+# container image tag, to see all available tags go to
+# https://github.com/srl-labs/protoc-container/pkgs/container/protoc
```

Comment:
Is this container something that you expect to continue to be updated?

Reply:
yes, I just bumped it to a more recent version tuple [227be07](https://github.com/openconfig/gnmi/pull/136/commits/227be073d1d6105d6e67548e74317bd7617a39a8)